.dockerignore

```diff
@@ -26,4 +26,19 @@ r
 **/junk
 **/scripts
 **/python_cache*
-**.venv
\ No newline at end of file
+**.venv
+**/*.zip
+**/*.grib2
+grib
+.vscode
+.github
+.ipynb_checkpoints
+.vscode
+**/__pycache__
+**/*.pyc
+python_cache
+junk/**/*
+junk
+scripts/**/*
+scripts
+*/nbconvert/*
```

Comment:
nit: A few duplicates in here unless the globs are missing something:
- .vscode
- python_cache
- scripts
- junk

Reply:
Thanks consolidated and reordered in: https://github.com/bcgov/wps/pull/4912/commits/1925349da001247f534a1d49a7df8e12f69145ff